python/cuml/dask/datasets/regression.py

```diff
@@ -211,12 +211,14 @@ def make_regression(n_samples=100, n_features=100, n_informative=10,
                                               chunks=(n_samples_per_part, -1))
 
     y = da.dot(X[:, :n_informative], ground_truth) + bias
+    X = X.rechunk((None, -1))
```

Comment:
Would be nice to see a small pytest to demonstrate this is actually yielding the expected shape in the end.